### PR TITLE
My first push request

### PR DESCRIPTION
**Pull request by Godfrey-Alimony**
You can comment here: your review.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<html>
```
Comment (Rustamxon7) [inline index.html:1]: # Congratulaton on your first pull request 👏🎉

- [ ] Welcome to microverse 🥳. Good job so far 👍.You did a great job so far but there are still some issues as given in the requirement. Please kindly fix the issues down below 👍


- Add a `.gitignore` file.
- Set up [a linter for HTML and CSS](https://github.com/microverseinc/linters-config) and make sure that any files that are not relevant are ignored by git.
- Add a descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.
- A test.md file with the text "This file should be ignored by git".
- Make sure that the last file is actually ignored by git and is not present in your GitHub repository.

**Review by Rustamxon7 — CHANGES_REQUESTED** [review]
Hi @Godfrey-Alimony , 

Good job so far!

### HIGHLIGHTS!

- No linters errors.💯
- Descriptive Readme.md. 💯


There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Godfrey-Alimony** [comment]
Added ingore file.

**Review by alexander16108 — COMMENTED** [review]
Hi @Godfrey-Alimony , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no linters checkers, gitignore file and a descriptive readme file 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._
```

**Pull request merged**